utilities/zookeeper/R/correctors.county.R

```diff
@@ -65,12 +65,13 @@ zyzzyva_county_corrections <- function(signals_list, ...){
   len_params <- sapply(params, length)
   max_len_params <- max(len_params)
   in_names <- names(signals_list[[1]])
-  assert_that(all(len_params %in% c(1L, max_len_params)),
-              msg = paste("In apply_corrections: ",
-                          "corrections parameters must be length 1 or the",
-                          "same length as the longest corrections parameter."))
+  assertthat::assert_that(all(len_params %in% c(1L, max_len_params)),
+                          msg = paste("In apply_corrections: ",
+                                      "corrections parameters must be length 1 or the",
+                                      "same length as the longest corrections parameter."))
   #Single Signal
-  if (params$signals_to_correct == "response") {
+  if (params$signals_to_correct == "confirmed_incidence_num" | 
+      params$signals_to_correct == "deaths_incidence_num") {
     corrected <- zyzzyva_county_corrections_single_signal(
       signals_list[[1]], params)
     signals_list[[1]] <- corrected %>%
```

Comment:
I'm still not quit sure I see how this will work. It seems like you're assuming that the first element of the `signals_list` will be one of those two signals. What if it's not?